enhancements/installer/nutanix-ahv-ipi.md

```diff
@@ -0,0 +1,167 @@
+---
+title: nutanix-ahv-ipi
+authors:
+  - "@vnephologist"
+reviewers:
+  - "@makentenza"
+  - "@fabianofranz"
```

Comment:
@staebler Who from the installer team should we list as a reviewer?
@JoelSpeed Who from the cluster infrastructure team should we list as a reviewer?

Reply:
you can list me as a reviewer, i'll help out